src/main.rs

```diff
@@ -50,16 +50,92 @@ fn main() -> GameResult {
 
 
 
+struct ClickableZone {
```

Comment:
<img src="https://github.com/alexesmet/nonogram-gamer-rs/assets/36900282/28ff5628-74b5-4bdd-9a63-97ddcc81ef0a" width="200" height="200">

IMHO deserves a separate file `clickable_zone.rs`

Reply:
Moved to clickable_zone.rs